ycmd/completers/go/tests/gocode_completer_test.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
```

Comment:
Thank you for writing tests!

I'd like to see one small test in `get_completions_test.py` for Go completion, just so that we have something that exercises the full path from HTTP request to HTTP response. It ensures we don't break the plumbing that connects the go semantic completer to the rest of ycmd.